source/CardGames.py

```diff
@@ -280,34 +280,125 @@ def winnnerCheck(self, house, player):
       return 1 if houseHand <= 21 else 2
 #####################################################################################################################
 """Initializations"""
+import pygame 
+  
+pygame.init() 
+  
+# screen size
+res = (1080,720) 
+  
+# opens up a window 
+screen = pygame.display.set_mode(res) 
+  
 
-player_name = input("Username: ")
+color = (255,255,255) 
+color_light = (170,170,170) 
+color_dark = (100,100,100) 
+  
+width = screen.get_width() 
+height = screen.get_height() 
 
-partition = player_name.find(";")
-config=""
-if partition > 0:
-  config = player_name[partition:]
-  player_name = player_name[0:partition]
-###
+#options
 deck_num = 1
-for_money = True
 early_shuffle = False
-###
-i=0
-label = ""
-while i < len(config):
-  if config[i] == ";":
-    label =""
-  else:
-    label = label + config[i]
-  i += 1
-  
-  if label == "free":
-    for_money = False
-  if label == "four":
-    deck_num = 4
-  if label == "shuffle":
-    early_shuffle = True
+for_money=True#unused
+
+
+##menu
+
+smallfont = pygame.font.SysFont('Corbel',35)
+
+#to add a new button only lines marked with ~~~ must be added
+quit_text = smallfont.render('QUIT' , True , color) #   ~~~
+decks_text = smallfont.render('decks' , True , color)
+play_text = smallfont.render('PLAY' , True , color)
+early_text = smallfont.render('shuffle' , True , color)
+
+buttonlist=[        #[left,up,width,hight,text_offset]
+    [   [480,500,120,35,20],quit_text ],#   ~~~
+    [   [480,350,120,35,20],decks_text ],
+    [   [480,100,120,35,20],play_text ],
+    [   [480,200,120,35,10],early_text ]
+]
+
+
+in_menu=True 
+while in_menu: 
+      
+    for ev in pygame.event.get(): 
+          
+        if ev.type == pygame.QUIT: 
+            pygame.quit() 
+              
+        #checks if a mouse is clicked 
+        if ev.type == pygame.MOUSEBUTTONDOWN: 
+              
```

Comment:
Because pygame isn't well known or known at all to some of the group members, it might be a good idea to add comments that explain some of pygame's functionality that is being used. This way working with the code might be more intuitive.

Reply:
comments added